extensions/amp-story-shopping/0.1/amp-story-shopping-attachment.js

```diff
@@ -105,45 +109,213 @@ export class AmpStoryShoppingAttachment extends AMP.BaseElement {
     }
     this.storeService_.subscribe(
       StateProperty.PAGE_ATTACHMENT_STATE,
-      (isOpen) => this.onPageAttachmentStateUpdate_(isOpen)
+      (isOpen) => {
+        const shoppingData = this.storeService_.get(
+          StateProperty.SHOPPING_DATA
+        );
+        this.checkClearActiveProductData_(isOpen, shoppingData);
+        this.updateTemplate_(isOpen, shoppingData);
+      }
+    );
```

Comment:
I'm no longer 100% sure about the order in which things are loaded. Is it possible that the user clicks to open the attachment before this code run? i.e. do we need to set `callToInitialize` to `true`?